build.go

```diff
@@ -61,30 +60,46 @@ func buildAction(clicontext *cli.Context) error {
 		"--output=type=docker,name="+tag)
 	buildctlCmd.Env = os.Environ()
 
-	// FIXME: do not rely on ctr command
-	ctrCmd := exec.Command("ctr",
-		"--address", containerdAddr,
-		"--namespace", ns,
-		"images",
-		"import",
-		"--snapshotter", sn,
-		"-")
-	ctrCmd.Env = os.Environ()
-
 	buildctlStdout, err := buildctlCmd.StdoutPipe()
 	if err != nil {
 		return err
 	}
 	buildctlCmd.Stderr = clicontext.App.Writer
-	ctrCmd.Stdin = buildctlStdout
-	ctrCmd.Stdout = clicontext.App.Writer
-	ctrCmd.Stderr = clicontext.App.ErrWriter
 
 	if err := buildctlCmd.Start(); err != nil {
 		return err
 	}
-	if err := ctrCmd.Start(); err != nil {
+
+	// Load images from buildkit tar stream
+	client, ctx, cancel, err := newClient(clicontext)
+	if err != nil {
 		return err
 	}
-	return ctrCmd.Wait()
+	defer cancel()
+
+	// FIXME: I wonder selecting the image digest type.
```

Comment:
What do you mean and who am "I"?

Reply:
I've forgotten.

Reply:
I'm terribly sorry for the unremoved comment.